packages-ui/roosterjs-react/lib/contextMenu/index.ts

```diff
@@ -15,4 +15,5 @@ export {
     TableEditShadeMenuItemStringKey,
     TableEditMenuItemStringKey,
     TableEditAlignTableMenuItemStringKey,
+    TableBorder_unfinished,
```

Comment:
Currently roosterjs-react is not supporting Content Model yet. So if possible, declare any UI related thing in demo site code only. Later we will do dedicate work on port roosterjs-react to use Content Model.